src/main/resources/log4j.properties

```diff
@@ -1,10 +1,16 @@
 # Root logger option
-log4j.rootLogger=DEBUG, stdout
+log4j.rootLogger=INFO, stdout, file
 
 # Redirect log messages to console
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.Target=System.out
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
 
-
+# Redirect log messages to a log file, support file rolling.
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.File=/jv-cinema-project/cinema_logs.log
```

Comment:
your `cinema_logs.log` should be in `logs` folder

Reply:
done